src/lwc/geFormField/geFormField.js

```diff
@@ -469,11 +481,17 @@ export default class GeFormField extends LightningElement {
         return siblingRecordTypeId ||
             this.parentRecordRecordTypeId() ||
             this.defaultRecordTypeId() ||
-            null;
+            undefined;
     }
 
     defaultRecordTypeId() {
-        return this.objectDescribeInfo && this.objectDescribeInfo.defaultRecordTypeId;
+        if(this.isTrueFalsePicklist) {
+            if(this.dataImportDescribeInfo) {
+                return this.dataImportDescribeInfo.defaultRecordTypeId;
+            }
+        } else {
+            return this.objectDescribeInfo && this.objectDescribeInfo.defaultRecordTypeId;
+        }
```

Comment:
Does this work need to support source object (DataImport) Record Types? This property generally represents the default RTId of the target object.  How will it be used when the source field is a picklist?

Reply:
We're rendering the actual picklist values from the Source Field whenever the source type is a picklist and the target is a checkbox.